src/vm.rs

```diff
@@ -150,9 +152,11 @@ pub struct UhyveVm<VirtBackend: VirtualizationBackend> {
 	pub virtio_device: Arc<Mutex<VirtioNetPciDevice>>,
 	#[allow(dead_code)] // gdb is not supported on macos
 	pub(super) gdb_port: Option<u16>,
+	pub(crate) mount: Mutex<UhyveFileMap>,
```

Comment:
see above: `mount` is a maybe misleading name

Reply:
I'm still in favor of `file_mapping`.

Reply:
This should be fixed.